src/ast_dumper.cpp

```diff
@@ -182,6 +200,34 @@ class OpGetter::OpGetterImpl : public NonModifyingVisitor {
     return op_;
   }
 
+  void Visit(const IncrExprNode&) override {
+    op_ = "++";
+  }
+
+  void Visit(const DecrExprNode&) override {
+    op_ = "--";
+  }
+
+  void Visit(const NegExprNode&) override {
+    op_ = "-";
+  }
+
+  void Visit(const AddrExprNode&) override {
+    op_ = "&";
+  }
+
+  void Visit(const DereferExprNode&) override {
+    op_ = "&";
```

Comment:
```suggestion
    op_ = "*";
```
Caught you on the "copy-paste" typo! :wink:
Time to ensure the test covers all of the terminals.